src/components/definition-list/definition-list.style.js

```diff
@@ -0,0 +1,70 @@
+import styled, { css } from 'styled-components';
+import { space } from 'styled-system';
+import { baseTheme } from '../../style/themes';
+import StyledButton from '../button/button.style';
+import LinkStyle from '../link/link.style';
+
+export const StyledDl = styled.dl`
+  display: inline-flex;
+  height: auto;
+  width: 100%;
+  background-color: transparent;
+  overflow: hidden;
+`;
+
+export const StyledDtDiv = styled.div`
+${space}
+  ${({ w, dtTextAlign }) => css`
+    text-align: ${dtTextAlign};
+    width: ${w}%;
+  `}
+`;
+
+export const StyledDdDiv = styled.div`
+  ${({ ddTextAlign }) => css`
+    text-align: ${ddTextAlign};
+    width: auto;
+  `}
+`;
+
+export const StyledDt = styled.dt`
+${space}
+  ${({ mb, pr, theme }) => css`
+    font-size: 14px
+    font-weight: 700;
+    padding-right: ${theme.spacing * pr}px;
+    color: ${theme.definitionList.dtTextDark};
+    margin-bottom: ${theme.spacing * mb}px;
```

Comment:
This should be removed as `mb` is handled by `${space}` above

Reply:
✅